Resources/ServerInfo/Guidebook/Rules/AlertProcedure.xml

```diff
@@ -0,0 +1,9 @@
+<Document>
+
+[color=#ff0000]Alert Procedure[/color]
+
+[color=#ff0000]MRP Only[/color]
+
+See https://wiki.spacestation14.io/wiki/Alert_Procedure
+
```

Comment:
Guidebook doesn't support tables and Alert Procedure is probably unreadable without it, same with Space Law.